examples/passwords.yml

```diff
@@ -10,3 +10,5 @@
 # $ ansible-playbook myplaybook.yml --ask-vault-pass
 #
 engine_password: 123456
+ovirt_repositories_rh_username: "myuser"
```

Comment:
When moving to `ovirt-ansible-repositories` repostiory, please remember to remove engine_password, as doesn't make sense to have it there.